build.py

```diff
@@ -840,6 +840,9 @@ def install_dcgm_libraries(dcgm_version, target_machine):
 
 
 def install_miniconda(conda_version, target_machine):
+    if target_machine == "arm64":
+        # Arm architecture on MacOS named arm64
+        target_machine = "aarch64"
```

Comment:
Conda does have a MacOS-arm64 build too. The reason that we need to change this line is because you are running Linux container on Mac? If so, can you update the comment to mention this.

Reply:
@Tabrizian yes, running linux container on mac, ok I will update it now